api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/NewCardService.java

```diff
@@ -3,23 +3,27 @@
 import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
+import uk.gov.dhsc.htbhf.claimant.factory.CardRequestFactory;
 import uk.gov.dhsc.htbhf.claimant.repository.ClaimRepository;
 
+import java.util.List;
+import java.util.UUID;
+
 @Service
 @AllArgsConstructor
 @Slf4j
 public class NewCardService {
 
+    private CardClient cardClient;
+    private CardRequestFactory cardRequestFactory;
     private ClaimRepository claimRepository;
 
-    /**
-     * Print out first 10 new claims (testing purposes only).
-     */
-    @Transactional
-    public void createNewCards() {
-        claimRepository.getNewClaims().limit(10).forEach(
-                claim -> log.info("Creating card for new claim {}", claim.getId())
-        );
+    public void createNewCards(List<UUID> claimIds) {
```

Comment:
This should really be dealing with a single claimId at a time in a @Transactional method and not be dealing with all the claims, it should be the responsibility of the Job and batching process to deal with the numbers of claims and handing them off to the ClaimService in the correct manner.